### DISCUSSION TITLE
I have created the task workflow using postman but I am not able to get these tasks into running

### DISCUSSION THREAD

**Discussion by ruchitadamodar1922** [Netflix/conductor#2549] [Q&A]
![image](https://user-images.githubusercontent.com/43370835/118688163-6d2d8d00-b823-11eb-9735-1592e40aca6c.png)

I have created the task workflow using postman but I am not  able to get these tasks into running

![image](https://user-images.githubusercontent.com/43370835/118688306-92220000-b823-11eb-8248-9e1badc9edb5.png)

**Comment by sudouser777** [comment]
@ruchitadamodar1922 can you share the postman request you made

**Comment by kishorebanala** [comment]
> @ruchitadamodar1922 can you share the postman request you made

+1. Please share the workflow definitions and payloads you've used to help reproduce this.

**Comment by ruchitadamodar1922** [comment]
@kishorebanala @komatiraju032  Sir These are the task defs and workflows in my postman
http://localhost:8080/api/metadata/taskdefs
[
    {
        "name": "add_product_to_catalogue",
        "retryCount": 3,
        "retryLogic": "FIXED",
        "retryDelaySeconds": 10,
        "timeoutSeconds": 300,
        "timeoutPolicy": "TIME_OUT_WF",
        "responseTimeoutSeconds": 180,
        "ownerEmail": "ruchitadamodar2000@gmail.com"
    },
    {
        "name": "delete_product_from_catatlogue",
        "retryCount": 3,
        "retryLogic": "FIXED",
        "retryDelaySeconds": 10,
        "timeoutSeconds": 300,
        "timeoutPolicy": "TIME_OUT_WF",
        "responseTimeoutSeconds": 180,
        "ownerEmail": "ruchitadamodar2000@gmail.com"
    },
    {
        "name": "get_all_products",
        "retryCount": 3,
        "retryLogic": "FIXED",
        "retryDelaySeconds": 10,
        "timeoutSeconds": 300,
        "timeoutPolicy": "TIME_OUT_WF",
        "responseTimeoutSeconds": 180,
        "ownerEmail": "ruchitadamodar2000@gmail.com"
    },
    {
        "name": "get_product_by_cat",
        "retryCount": 3,
        "retryLogic": "FIXED",
        "retryDelaySeconds": 10,
        "timeoutSeconds": 300,
        "timeoutPolicy": "TIME_OUT_WF",
        "responseTimeoutSeconds": 180,
        "ownerEmail": "ruchitadamodar2000@gmail.com"
    },
    {
        "name": "get_product_by_Id",
        "retryCount": 3,
        "retryLogic": "FIXED",
        "retryDelaySeconds": 10,
        "timeoutSeconds": 300,
        "timeoutPolicy": "TIME_OUT_WF",
        "responseTimeoutSeconds": 180,
        "ownerEmail": "ruchitadamodar2000@gmail.com"
    },
    {
        "name": "get_product_by_Name",
        "retryCount": 3,
        "retryLogic": "FIXED",
        "retryDelaySeconds": 10,
        "timeoutSeconds": 300,
        "timeoutPolicy": "TIME_OUT_WF",
        "responseTimeoutSeconds": 180,
        "ownerEmail": "ruchitadamodar2000@gmail.com"
    }
]
http://localhost:8080/api/metadata/workflow/
{
    "name": "product_catalogue_service3",
    "description": "Creating and maintaing the catalogue of all the products that the user can view",
    "ownerEmail":"ruchitadamodar2000@gmail.com",
    "version": 2,
    "schemaVersion": 2,
    "tasks":
    [
        {
            "name": "decide_controller",
            "taskReferenceName": "decide",
            "inputParameters": {
                "controllerId": "${workflow.input.controller}"
            },
            "type": "DECISION",
            "caseValueParam": "case_value_param",
            "decisionCases": {
            "1": [
            {
                "name": "AdminTask",
                "taskReferenceName": "decide2",
                "inputParameters": {
                "value": "${workflow.input.value}"
                },
                "type": "DECISION",
                "caseValueParam": "case_value_param",
                "decisionCases": 
                {
                    "add_product_to_catalogue":[
                    {
                        "name": "add_product_to_catalogue",
                        "taskReferenceName": "ad1",
                        "inputParameters": {
                        "productId": "${workflow.input.productId}"
                        },
                        "type": "SIMPLE" 
                    }],
                    "delete_product_from_catatlogue":[
                    {
                    "name": "delete_product_from_catatlogue",
                    "taskReferenceName": "dp1",
                    "inputParameters": {
                    "productId": "${workflow.input.productId}"
                    },
                    "type": "SIMPLE"
                    }]
                }
            }
        ],
        "2": [
        {
            "name": "ProductController",
            "taskReferenceName": "decide3",
            "inputParameters": {
                "value": "${workflow.input.value}"
            },
            "type": "DECISION",
            "caseValueParam": "case_value_param",
            "decisionCases":
            {
                "get_all_products":[
                {
                    "name":"get_all_products",
                    "taskReferenceName": "gap",
                    "inputParameters": {
                        "movieId": "${workflow.input.productId}"
                    },
                    "type": "SIMPLE"
                }],
                "get_product_by_cat":[
                {
                    "name":"get_product_by_cat",
                    "taskReferenceName": "gpbc",
                    "inputParameters": {
                        "movieId": "${workflow.input.productcatId}"
                    },
                    "type": "SIMPLE"
                }],
                "get_product_by_Id":[
                {
                    "name":"get_product_by_Id",
                    "taskReferenceName": "gpbI",
                    "inputParameters": {
                    "movieId": "${workflow.input.productId}"
                    },
                    "type": "SIMPLE"
            }],
            "get_product_by_Name":[
            {
                "name":"get_product_by_Name",
                "taskReferenceName": "gpbn",
                "inputParameters": 
                {
                "movieId": "${workflow.input.productName}"
                },
                "type": "SIMPLE"
            }]
            }
        }
    ]
}
        }
    ]
}
http://localhost:8080/api/tasks/poll/decide_controller/
http://localhost:8080/api/tasks/
{
  "workflowInstanceId": "a42a69e9-3181-41c9-9393-5a8090d3ba76",
  "taskId": "336f7cd5-ae3a-4529-9c1f-ba5d44d997e9",
  "reasonForIncompletion": "",
  "callbackAfterSeconds": 0,
  "workerId": "Marcos-Worker-2020",
  "status": "COMPLETED"
}
http://localhost:8080/api/workflow/
{
    "name": "product_catalogue_service3",
    "version": 2,
    "correlationId": "Product_catalogue_service3",
    "input": {
         "controllerId":1,
         "value":"add_product_to_catalogue",
        "productId":"1234"     
    }
}

**Comment by apanicker-nflx** [comment]
what is the output of this request - 
> http://localhost:8080/api/workflow/
{
"name": "product_catalogue_service3",
"version": 2,
"correlationId": "Product_catalogue_service3",
"input": {
"controllerId":1,
"value":"add_product_to_catalogue",
"productId":"1234"
}
}

**Comment by kishorebanala** [comment]
@ruchitadamodar1922 Also, if your workflows are started successfully, but are not visible in the UI.. It is most likely related to how Elasticsearch is configured. Do you mind sharing your Conductor server set-up please. Can you also double check few other things, such as Elasticsearch status, if the workflow and task data is being written successfully to Elasticsearch, if they can be retrieved successfully etc.
